sdk/hardwaresecuritymodules/Azure.ResourceManager.HardwareSecurityModules/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please provide the release date by replacing the `Unreleased`, format is `yyyy-mm-dd`

Reply:
Updated the release date, can we make this RP as Do not merge, please ?